mephisto/abstractions/providers/mturk/mturk_utils.py

```diff
@@ -67,15 +67,39 @@ def setup_aws_credentials(
         if register_args is not None:
             # Eventually we could manually re-parse the file and see
             # if the credentials line up or not, then fix ourselves
-            print(
-                f"WARNING credentials provided, but there's already a "
-                f"profile for {profile_name}. If these don't line up, you'll "
-                f"need to manually navigate to your ~/.aws/credentials file "
-                f"and remove the entry for this profile name, then run again.\n"
-                f"As this profile is currently loading, we consider it "
-                f"successfully registered anyways."
-            )
+            aws_credentials_file_path = "~/.aws/credentials"
+            expanded_aws_file_path = os.path.expanduser(
+                aws_credentials_file_path)
+            aws_credentials_file_string = ""
+            if os.path.exists(expanded_aws_file_path):
+                with open(expanded_aws_file_path, "r") as aws_credentials_file:
+                    aws_credentials_file_string = aws_credentials_file.read()
+            # accessing the aws_credentials_file
+            aws_access_key_id = aws_credentials_file_string.split("\n")[1]
```

Comment:
This won't get you the access key ID always, as the access key ID isn't always on the first line. If you add multiple profiles, there may be many lines to search through, and it will be hard to select the correct one by directly indexing.

Your best bet here is to do a regex search for:
```
"[<profile_name>]\naws_access_key_id=$1\naws_secret_access_key=$2\n
```
And see if the values at $1 and $2 match the given `aws_access_key_id` and `aws_secret_access_key`. If they match, you can just return True. If they don't, you can do a `aws_credentials_file_string.replace(<found_version>, <new_version>)`.